hystrix/src/main/java/feign/hystrix/HystrixInvocationHandler.java

```diff
@@ -62,6 +63,8 @@ protected Object run() throws Exception {
 
     if (HystrixCommand.class.isAssignableFrom(method.getReturnType())) {
       return hystrixCommand;
+    } else if (Observable.class.isAssignableFrom(method.getReturnType())) {
+      return hystrixCommand.toObservable();
```

Comment:
Good point. I returned a "cold" Observable to allow the consumer more control over execution and it definitely needs to be documented.

Is there a case to make this configurable or return a hot observable instead? One advantage of a "hot" observable is that you don't need to subscribe if you're not interested in the result. This might be the case when performance an update style request.